lib/wda/utils.js

```diff
@@ -71,7 +71,7 @@ async function checkForDependencies (bootstrapPath, useSsl = false) {
       'and make sure the root folder, where carthage binary is installed, is present in PATH environment variable. ' +
       `The current PATH value: '${process.env.PATH ? process.env.PATH : "<not defined for the Appium process>"}'`);
   }
-  const carthageRoot = `${bootstrapPath}/Carthage`;
+  const carthageRoot = `${bootstrapPath}/${CARTHAGE_ROOT}`;
```

Comment:
This should probably be `path.resolve` as well (like line 283 below)

Reply:
changed